### DISCUSSION TITLE
Uyouplus Question

### DISCUSSION THREAD

**Discussion by ImpenetrableNoble** [qnblackcat/uYouPlus#1069] [Q&A] [answered]
Hi Uyouplus Team, I wondering if this possible Uyouplus mobile application is able to be associated with other app called TestFlight from apple's App Store? Perhaps Uyouplus users would find it easier to provide feedback to you guys easily if it's from the TestFlight application.

**Comment by yodaluca23** [comment, marked as answer]
I wish LOL, but theirs no way Apple would approve it..
